### PR TITLE
fastd: add page

### PR DESCRIPTION
**Pull request by marek22k**
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** v22

### CODE REVIEW

File: pages/common/fastd.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# fastd
+
+> VPN daemon.
+> Works on Layer 2 or Layer 3, supports different encryption methods, used by Freifunk.
+> More information: <https://fastd.readthedocs.io/en/stable/>.
+
+- Start fastd with a specific configuration file:
+
+`fastd --config {{fastd.conf}}`
```
Comment (CleanMachine1) [inline pages/common/fastd.md:9]:
```suggestion
`fastd --config {{path/to/fastd.conf}}`
```


Could apply this throughout please.

Comment (marek22k) [reply pages/common/fastd.md]: Thanks for the hint!

File: pages/common/fastd.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+> VPN daemon.
+> Works on Layer 2 or Layer 3, supports different encryption methods, used by Freifunk.
+> More information: <https://fastd.readthedocs.io/en/stable/>.
+
+- Start fastd with a specific configuration file:
+
+`fastd --config {{path/to/fastd.conf}}`
+
+- Start a Layer 3 VPN with an MTU of 1400, load the rest of the configuration parameters from a file:
```
Comment (CleanMachine1) [inline pages/common/fastd.md:11]:
```suggestion
- Start a Layer 3 VPN with an MTU of 1400 loading the rest of the configuration parameters from a file:
```

**Review by CleanMachine1 — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by blueskyson — APPROVED** [review]

**Pull request merged**